packages/keys/src/key-pair.ts

```diff
@@ -9,3 +12,14 @@ export async function generateKeyPair(): Promise<CryptoKeyPair> {
     );
     return keyPair as CryptoKeyPair;
 }
+
+export async function createKeyPairFromBytes(bytes: Uint8Array, extractable?: boolean): Promise<CryptoKeyPair> {
+    if (bytes.byteLength !== 64) {
+        throw new SolanaError(SOLANA_ERROR__INVALID_KEYPAIR_BYTES);
```

Comment:
Is it worth adding stuff to the context, like the actual length, or the actual bytes? Maybe this is where we start needing ‘private’ context properties (ie. ones that won't get serialized by the context serializer).

Reply:
Happy to add that info to the context. Is it really an attack vector to expose the length of the provided bytes though? I think a private context property might be a bit of a stretch here.